service/sensor.go

```diff
@@ -26,12 +28,17 @@ func CreateSensor(w http.ResponseWriter, r *http.Request) {
 		respondWithError(w, http.StatusInternalServerError, err.Error())
 		return
 	}
-	sensor.ID = bson.NewObjectId()
-	if err := sensorRepository.InsertSensor(sensor); err != nil {
-		respondWithError(w, http.StatusBadRequest, "Invalid request")
-		return
+	sensor.Url = strings.ToLower(sensor.Url)
+	if statusUrl := verifyIntegrityUrlSensor(sensor.Url); statusUrl != false {
+		sensor.ID = bson.NewObjectId()
+		if err := sensorRepository.InsertSensor(sensor); err != nil {
+			respondWithError(w, http.StatusBadRequest, "Invalid request")
+			return
+		}
+		respondWithJson(w, http.StatusOK, map[string]string{"result": "sucess"})
```

Comment:
@adrissonGalvao created objects should return http status "Created" instead "OK".
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes